packages/ckeditor5-alignment/docs/features/text-alignment.md

```diff
@@ -11,6 +11,16 @@ The {@link module:alignment/alignment~Alignment} feature enables support for tex
 
 {@snippet features/text-alignment}
 
+## Related features
+
+There are more CKEditor 5 features that can help you organize your content:
+* {@link features/basic-styles Basic font styles} &ndash; The essentials, like **bold**, *italic* and others.
```

Comment:
I'm not sure this one is related. Text alignment works on block level and sort of adds the structure. All other features are from the same kind, basic styles look off to me here. If you don't want to remove it, maybe it should land at the end?